go.mod

```diff
@@ -4,6 +4,7 @@ go 1.15
 
 require (
 	github.com/cloudflare/cfssl v1.5.0 // for `kubebuilder alpha config-gen`
+	github.com/go-logr/logr v0.3.0 // indirect
```

Comment:
Try deleting these 3 imports and check if `go mod tidy` adds them.

Reply:
Running the tests does, but building the binary doesn't.  Will delete.